backend/battle/views.py

```diff
@@ -106,3 +128,119 @@ class BattleSignUp(CreateView):
 
 class SignUpSucess(TemplateView):
     template_name = "battle/user/sucess_signup.html"
+
+
+class PasswordContextMixin:
+    extra_context = None
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update({
+            'title': self.title,
+            **(self.extra_context or {})
+        })
+        return context
+
+
+class PasswordCreateConfirmView(PasswordContextMixin, FormView):
```

Comment:
You don't need to copy the whole code. You can make a new class, inherit from the existing one, and change only what you need.

Reply:
Thank you!